common/smoke-tests/pom.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>smoke-test</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
nit: This version doesn't match our versioning scheme <major>.<minor>.<hotfix> or <major>.<minor>.<hotfix>-**beta**.X.